op-service/flags/flags.go

```diff
@@ -0,0 +1,83 @@
+package flags
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/urfave/cli/v2"
+
+	"github.com/ethereum-optimism/optimism/op-node/chaincfg"
+	opservice "github.com/ethereum-optimism/optimism/op-service"
+)
+
+const EnvVarPrefix = "SUPERCHAIN"
+
+var (
+	RollupConfig = &cli.StringFlag{
+		Name:    "superchain.rollup.config",
+		Usage:   "Rollup chain parameters",
+		EnvVars: opservice.PrefixEnvVar(EnvVarPrefix, "ROLLUP_CONFIG"),
+	}
+	Network = &cli.StringFlag{
+		Name:    "superchain.network",
+		Usage:   fmt.Sprintf("Predefined network selection. Available networks: %s", strings.Join(chaincfg.AvailableNetworks(), ", ")),
+		EnvVars: opservice.PrefixEnvVar(EnvVarPrefix, "NETWORK"),
+	}
+	CanyonOverrideFlag = &cli.Uint64Flag{
+		Name:    "superchain.override.canyon",
+		Usage:   "Manually specify the Canyon fork timestamp, overriding the bundled setting",
+		EnvVars: opservice.PrefixEnvVar(EnvVarPrefix, "OVERRIDE_CANYON"),
+		Hidden:  false,
+	}
+	DeltaOverrideFlag = &cli.Uint64Flag{
+		Name:    "superchain.override.delta",
+		Usage:   "Manually specify the Delta fork timestamp, overriding the bundled setting",
+		EnvVars: opservice.PrefixEnvVar(EnvVarPrefix, "OVERRIDE_DELTA"),
+		Hidden:  false,
+	}
+)
```

Comment:
Moved rollup cfg related flags to `op-service` package which could be used across services, kept the original ones inside `op-node` to allow no change on client side

Reply:
See my note above.